stdlib/callstack.mli

```diff
@@ -0,0 +1,139 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Call stacks.
+
+    Facilities for getting the current call stack, printing and inspecting them.
+
+    @since 4.08 *)
+
+type t
+(** The abstract type [t] stores a callstack.
```

Comment:
There's a mix of "callstack" and "call stack" in the text, one should be chosen. Maybe this suggests it should rather be `Call_stack`; stack already appears separately in `Stack_overflow`.

Reply:
Backtrace is less clear than "call stack" or "stack trace" to me. I don't mind if you want to call it `Callstack` and if you use "call stack" in the text but at least do it consistently. 

That said if people never use "callstack" as a single word, conventions would say that `Call_stack` is the right identifier. 

You could also call it `Stack_trace` which would, terminlogy-wise, blend better with `Stack_overflow` exception.

Reply:
OK, let's go with `Stack_trace`. I pushed a commit with the renaming.